docs/src/groupby.md

```diff
@@ -61,7 +61,6 @@ accident_counts = accident_list.groupby("latimes_make_and_model").size().reset_i
 ```
 
 You can clean up the `0` column name assigned by pandas with the `rename` method. The `inplace` option, found on many pandas methods, will save the change to your variable automatically.
-When you execute a cell, the code or text inside it will be processed and the output will be displayed below the cell. If the cell contains code, it will be executed by the kernel (the Jupyter notebook's underlying programming language, such as Python) and any output generated by the code will be displayed.
```

Comment:
Good cut. This does seem like something we covered earlier in the class.

Reply:
Yep. It's in the notebook section. I flagged it to move it over and then noticed it was already there.